src/views/views.py

```diff
@@ -71,13 +71,27 @@ def home(request):
         order_by = filters.get('order_by') == 'asc' if '' else '-'
         job_offers = job_offers.order_by(f'{order_by}due_date')
 
+    query = filters.get('q')
+    if query:
+        job_offers = job_offers.filter(title__icontains=query)
+
     return GeneralViewController(request).render('pages/home.html', {
         'job_offers': job_offers,
         'companies_list': companies_list,
         'company_filter': filters.get('company'),
         'order_filter': filters.get('order_by'),
+        'query': query,
     })
 
+def home_search(request):
```

Comment:
@Dagurf21 what's this tho?

Reply:
Started doing a new view for the search handling, didnt need it just added it to def(home) view. Can be deleted

Reply:
Can you delete it so I can merge this? 🙏